src/Pages/AdminHomePage.js

```diff
@@ -1,43 +1,122 @@
-import React from 'react';
-import {Link} from 'react-router-dom';
+import React,{Component} from 'react';
 import Button from 'react-bootstrap/Button';
-function AdminHomePage() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        
-        <Link to='/CreateSchool'>
-            <Button variant="outline-danger">
-              Create School
-            </Button>
-        </Link>
-          
-        
-        <br />
-        <Link to='/CreateClassroom'>
-            <Button variant="outline-info">
-            Create Classroom
-            </Button>
-        </Link>
+import Table from 'react-bootstrap/Table';
+import InputGroup from 'react-bootstrap/InputGroup';
+import FormControl from 'react-bootstrap/FormControl';
+import {Link} from 'react-router-dom'
 
-        <br />
-
-        <Link to='/AddStudent'>
-            <Button variant="outline-warning">
-            Add Student
-            </Button>
-        </Link>
-
-        
+class SchoolTable extends Component{
+    render(){
+        const {data,headers} = this.props;
+        return(
+            <Table striped bordered hover>
+            <thead>
+              <tr>
+                {headers.map((header,index)=>{
+                    return(<td key={index}>{header}</td>)
+                })}
+              </tr>
+            </thead>
+            <tbody>
+              {Object.keys(data).map((key,index)=>{
+                  return(
+                    <tr key={index}>
+                        <td>{key}</td>
+                        <Link to='/createclassroom'>
+                            <td>{data[key].schoolname}</td>
+                        </Link>
+                        <td>{data[key].strength}</td>
+                    </tr>
+                  )
+              })}
+            </tbody>
+          </Table>
+        )
+    }
+}
+class  AdminHomePage extends Component {
+    componentDidMount(){
+        const schoolsPresent = require('./mockdata/schooldata.json');
+        this.setState({schoolsPresent})  
+    }
+    constructor(props){
+        super(props);   
+        this.state={
+            schoolsPresent:{} , 
+            newschool:{
+                name:'',
+                strength:''
+            }
+        }
+    }
+    createschool (){
+        let newid = 'sid'+(Object.keys(this.state.schoolsPresent).length+1);
```

Comment:
Later on, the logic of assigning an ID can be removed because Id will be automatically assigned by the database.